csharp_webapi/11.QnAMaker/QnAMaker/AppInsights/MyAppInsightsLoggerMiddleware.cs

```diff
@@ -0,0 +1,308 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Per our discussion today, please remove app insights from this sample and move it to be its own sample